sql/ GenrePreference.sql

```diff
@@ -0,0 +1,17 @@
+SELECT CASE
+           WHEN DATEDIFF(YEAR, Borrowers.DateOfBirth, GETDATE()) BETWEEN 0 AND 10 THEN '0-10'
```

Comment:
this doesn't take into account the month and day, depending on the requirements, this could give inaccurate results.

Reply:
Since the goal is to group borrowers by common age ranges (like "0-10 years," "11-20 years," etc.),
I used **DATEDIFF** to Return the difference between two date values, in years 

if we should group by age on moths we can use the same DATEDIFF to Return the difference between two date values, in Month like the following: 

`           WHEN DATEDIFF(MONTH, Borrowers.DateOfBirth, GETDATE()) BETWEEN 0 AND 10 THEN '0-10'
`